src/lib/stepper/stepper-vertical.html

```diff
@@ -0,0 +1,37 @@
+<div class="mat-stepper-container">
+    <div *ngFor="let step of _steps; let i = index" class="mat-step" role="tablist">
+        <div #stepHeader class="mat-step-header" role="tab"
+             [id]="_getStepLabelId(i)"
+             [attr.aria-controls]="_getStepContentId(i)"
+             [attr.aria-selected]="selectedIndex == i"
+             [tabIndex]="_focusIndex == i ? 0 : -1"
+             (click)="select(step)"
+             (keydown)="_onKeydown($event)">
+            <div *ngIf="step.active" class="active-step">{{i + 1}}</div>
+            <div *ngIf="!step.active" class="inactive-step">{{i + 1}}</div>
```

Comment:
Seems like you could use ngClass here, given that that's the only thing changing on the conditional.

```html
<div [ngClass]="{'active-step': step.active, 'inactive-step': !step.active}">{{ i + 1}} </div>
```